drivers/media/platform/bcm2835/bcm2835-unicam.c

```diff
@@ -0,0 +1,2372 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * BCM2835 Unicam capture Driver
+ *
+ * Copyright (C) 2017-2020 - Raspberry Pi (Trading) Ltd.
+ *
+ * Dave Stevenson <dave.stevenson@raspberrypi.org>
+ *
+ * Based on TI am437x driver by Benoit Parrot and Lad, Prabhakar and
```

Comment:
Indian name - Prabhakar Lad.
Amend to 
```
* Based on TI am437x driver by 
*   Benoit Parrot <bparrot@ti.com>
*   Lad, Prabhakar <prabhakar.csengg@gmail.com>
 ```

Likewise the line below ```Benoit Parrot, <bparrot@ti.com>```

Reply:
Done